src/reusableComponents/NotificationPopup/index.jsx

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+
+function NotificationPopup({ requests, acceptRequest, rejectRequest }) {
+  return (
+    <div className="notification-popup">
+      <ul>
```

Comment:
there's no solution for no notifcation. In case of no notifcation display something like no notification left to show